rust/cloud-storage/system_properties/src/repository/email_attachment.rs

```diff
@@ -0,0 +1,146 @@
+//! Email attachment property types and operations.
```

Comment:
@danielkweon can you follow the hex examples we have for things like `teams` crate and `roles_and_permissions` and `soup` to correctly use the trait pattern for repository/service so we can more easily add tests for this stuff in the future